crm_phonecall_planner/wizards/crm_phonecall_planner.py

```diff
@@ -205,7 +205,6 @@ def _schedule_call(self, partner, when):
                 "partner_id": partner.id,
                 "partner_phone": partner.phone,
                 "source_id": self.source_id.id,
-                "tag_ids": [(6, 0, self.tag_ids.ids)],
```

Comment:
What you have to change is the comodel in the field for this wizard as the relation nama has changed from `crm.lead.tag` to `crm.tag`.